extraConfigSriov.py

```diff
@@ -152,8 +155,12 @@ def ExtraConfigSriovSubscription(cc: ClustersConfig, cfg: ExtraConfigArgs, futur
     client.oc("create -f manifests/nicmode/sriov-namespace-config.yaml")
     client.oc("create -f manifests/nicmode/sriov-operator-group.yaml")
     client.oc("create -f manifests/nicmode/sriov-subscription.yaml")
+    version = cc.version
 
-    client.wait_for_crd(name="default", cr_name="sriovoperatorconfig", namespace="openshift-sriov-network-operator")
+    if parse_version_into_tuple(version) < parse_version_into_tuple("4.16.0"):
```

Comment:
Can we just check for the pods to be created in version < 4.16 as well? It would be better to keep this consistent if possible